app/components/TableWrapper.tsx

```diff
@@ -43,17 +46,21 @@ export default function TableWrapper<T extends Record<string, unknown>>({
 
   const query = useQueryBuilder(queryConfig);
 
-  const { tableName } = queryConfig;
-
-  const { data, isLoading } = useQuery<Record<typeof tableName, T[]>>({
+  const { data, isLoading, error } = useQuery({
```

Comment:
if you mouseover `data` in your code editor, you will see that it is typed as `T[]`. It's an array of the type that is coming from our trust zod schema 🔗